pages/report-hazard/style.css

```diff
@@ -1,3 +1,55 @@
+/* loadign animation */
+.lds-hourglass {
+  display: inline-block;
```

Comment:
Let's you move this loader to a different component.

Reply:
Done